interp: Add --value-output/-V option to do tovalue before output (#385)
wader/fq: doc/usage.md

### CLI arguments

> Most of jq's CLI argument work with fq. But here are some additional ones specific to fq:

Reviewer: ```suggestion
Most of jq's CLI arguments work with fq. But here are some additional ones specific to fq:
```

Start interactive REPL.

> Can be used with no input, one and multiple inputs, ex just `fq -i ` start a REPL with `null` input, `fq -i 123` with the number 123 as input, `fq -i . a b` with two files as input. Also works with `--slurp`. In the REPL it is also possible to start a sub-REPL:s by ending a query with `<query> | repl`, use ctrl-D to exit the sub-REPL. The sub-REPL will evaluate separately on each output from the query it was started. Use `[<query>] | repl` if you want to "slurp" into an array.

Reviewer: ```suggestion
Can be used with no input, one and multiple inputs, for example `fq -i ` starts a REPL with `null` input, `fq -i 123` with the number 123 as input, `fq -i . a b` with two files as input. This also works with `--slurp`. In the REPL it is also possible to start a sub-REPLs by ending a query with `<query> | repl`, use ctrl-D to exit the sub-REPL. The sub-REPL will evaluate separately on each output from the query it was started. Use `[<query>] | repl` if you want to "slurp" into an array.
```
